frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -471,8 +472,10 @@ class SharingSidebar extends React.Component {
             {({ onClose }) => (
               <DeleteModalWithConfirm
                 objectType="pulse"
-                title={t`Archive` + ' "' + pulse.name + '"?'}
-                buttonText={t`Archive`}
+                title={
+                  t`Delete this subscription to ` + ' "' + pulse.name + '"?'
```

Comment:
```suggestion
                  t`Delete this subscription to ${pulse.name}?`
```

Since this is a template you can use `${var}` instead of doing string concatenation. One thing I don't remember for sure though is whether you need to use `jt` instead of `t` for that. @paulrosenzweig mind refreshing my memory?

Reply:
Ah good call. And pretty sure you're right that we need to use `jt` here; this is the example in the developer's guide:

`const someJSX = <div>{jt`Hello ${name}`}</div>;`

Reply:
I think we're good with `t` here. `jt` lets you use jsx within the string. 

```js
t`Delete this subscription to ${pulse.name}?`;

const styledName = <span className="funky-style">{pulse.name}<span>;
jt`Delete this subscription to ${styledName}?`;
```